### PR TITLE
Update Textexpander to v6.1.2

### PR DESCRIPTION
**Pull request by joshmedeski**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/textexpander.rb [hunk 1]
```diff
@@ -10,12 +10,10 @@
 
     url "https://cdn.smilesoftware.com/TextExpander_#{version}.zip"
   else
-    version '514.0-1456193896'
-    sha256 'a299cdd809c3a5822375eec18a4644a785d7c442d6357a3a6f141c2e2b92a657'
+    version '6.1.2'
+    sha256 '744a2b1dfef3d3d4a71dd045094152ea1c7fbd2e901c7d87cc752f5b21ec3887'
 
-    url "https://dl.smilesoftware.com/com.smileonmymac.textexpander/#{version.sub(%r{-.*}, '')}/#{version.sub(%r{.*-}, '')}/TextExpander-#{version.sub(%r{-.*}, '')}.zip"
-    appcast 'https://updates.smilesoftware.com/com.smileonmymac.textexpander.xml',
```
Comment (reitermarkus) [inline Casks/textexpander.rb]: Did you check if the new version also has an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md)?

Comment (joshmedeski) [inline Casks/textexpander.rb]: I spent some time looking but couldn't find the appcast :(

Comment (joshmedeski) [inline Casks/textexpander.rb]: Ok, I think I found the proper information 😄

Comment (reitermarkus) [inline Casks/textexpander.rb]: Thanks!

Comment (adidalal) [inline Casks/textexpander.rb]: New appcast should actually be: `https://smilesoftware.com/appcast/TextExpander6.xml`

(from https://smilesoftware.com/appcast/update.php?appName=TextExpander&language=en&appVersion=6.1.2&bundleVersion=612.0)

**Review by adidalal — CHANGES_REQUESTED** [review]
Please update the appcast.

**Pull request merged**